Source/Rhetos.Dsl/DslParser.cs

```diff
@@ -86,12 +86,12 @@ protected IEnumerable<IConceptParser> CreateGenericParsers()
 
             _keywordsLogger.Trace(() => string.Join(" ", conceptMetadata.Select(cm => cm.conceptKeyword).OrderBy(keyword => keyword).Distinct()));
 
-            var result = conceptMetadata.Select(cm => new GenericParser(cm.conceptType, cm.conceptKeyword)).ToList<IConceptParser>();
+            var result = conceptMetadata.GroupBy(cm => cm.conceptKeyword.ToLower()).ToDictionary(x => x.Key, x => x.Select(cm => (IConceptParser)new GenericParser(cm.conceptType, cm.conceptKeyword)).ToList());

```

Comment:
`MultiDictionary` and `GroupBy` should both use case insensitive comparer (OrdinalIgnoreCase), so that `ToLower` method can be removed.